pages/common/ln.md

```diff
@@ -6,9 +6,13 @@
 
 `ln -s {{path/to/original/file}} {{path/to/link}}`
 
+- overwrite a symbolic link to a file
+
+`ln -sf {{path/to/new/original/file}} {{path/to/file/link}}`
+
 - overwrite a symbolic link to a folder
 
-`ln -sfT {{path/to/new/original/file}} {{path/to/link}}`
+`ln -sfT {{path/to/new/original/file}} {{path/to/folder/link}}`
```

Comment:
Doesn't the `-s` option make sure the link is symbolic? Below you say it's hard. Just trying to get things cleared up.